app/jsx/components/MediaFileGridComp.jsx

```diff
@@ -14,20 +22,16 @@ class CellComp extends React.Component {
         fileLabel = m[mimeSimple] ? m[mimeSimple] : p.file 
     return (
       <div className={"o-mediafile--" + mimeSimple}>
-        {/* Some items have no titles but use descriptions instead, and these get moved into title.
-            Let's make these look better (and load faster) by avoiding dotdotdot in this case only.
-        */}
-        <h2 className="o-mediafile__heading" ref={ el => { if ((p.useFilenameForTitle) || (p.description && p.description.length > 0)) $(el).dotdotdot({watch:"window"}) } }>
-          <Link to={url} target="_blank">{p.title}</Link>
-        </h2>
-        <Link className="o-mediafile__link" to={url} aria-label={fileLabel} target="_blank">
+        <h2 className="o-mediafile__heading" ref={ el => { if (p.title && p.title.length > 38) $(el).dotdotdot({watch:"window"}) } }>{p.title}</h2>
+        <div className="o-mediafile__preview" onClick={() => {this.view(url)}} aria-label={fileLabel} target="_blank">
```

Comment:
This will eventually pop up a modal but Joel isn't done building it.

Reply:
Ok